### PR TITLE
Support Kytea python3

### LINKED ISSUE #2: Support kytea with python3
Thank you for including Mykytea! I supported [Mykytea-python for Python3](https://pypi.python.org/pypi/kytea/0.1.3). Could you support kytea with Python 3?

I tried to implement python3 version, but I can't install jctconv https://github.com/ikegami-yukino/jctconv/issues/3 , so I haven't create patch yet.

### PR DESCRIPTION
**Pull request by chezou**
This PR makes to use Kytea with Python3.

close #2

### CODE REVIEW

File: JapaneseTokenizer/datamodels.py [hunk 1]
```diff
@@ -43,17 +43,19 @@ def convert_list_object(self):
 
 
     class TokenizedResult(object):
-        def __init__(self, analyzed_line, tuple_pos, word_stem, word_surface, is_feature=True, is_surface=False):
-            assert isinstance(analyzed_line, str)
+        def __init__(self, analyzed_line, tuple_pos, word_stem, word_surface, is_feature=True, is_surface=False, misc_info=None):
+            assert isinstance(analyzed_line, (str, type(None)))
```
Comment (chezou) [inline JapaneseTokenizer/datamodels.py]: BTW, I think parseToNode is works fine for python3 https://github.com/taku910/mecab/pull/24 , but I don't know whether mecab-python3 supports it.

**Comment by Kensuke-Mitsuzawa** [comment]
Thank you for your PR. I checked. I think.... you might forget putting some files in your commit.

Considering your changes, following files should exist, however they are not in your PR.

`JapaneseTokenizer/kytea_wrapper/kytea_wrapper_python3.py`
and
`test/test_kytea_wrapper_python3.py`

Are they just copied one of python2 version? Or did you add special process for python3?

**Comment by chezou** [comment]
Oops... I added forgotten files.

**Comment by Kensuke-Mitsuzawa** [comment]
Thanks ;)

**Pull request merged**